app/models/spree/taxjar.rb

```diff
@@ -176,14 +171,18 @@ def address_params
           to_country: tax_address_country_iso,
           to_zip: tax_address_zip,
           to_state: tax_address_state_abbr,
-          to_city: tax_address_city
+          to_city: tax_address_city,
+          from_country: 'US',
```

Comment:
?? what's this? this was hardcoded in my app. I think I asked not to use the last commit.